test_train.py

```diff
@@ -0,0 +1,158 @@
+#BSD 3-Clause License
+#
+#Copyright (c) 2022, AI2 Climate Modeling Team
```

Comment:
Not sure how copyrighting works for us.

Reply:
There is no need to have a separate license in each file, so I'd just leave this whole section out and defer to the main repo license. We could add a copyright notice there (in a different PR). We would add it in addition to leaving in place the copyright of the original authors.

Reply:
👍